samples/simulation/h2/command-line/Host.cs

```diff
@@ -35,7 +35,7 @@ static void Main(string[] args)
 
                 // To call a Q# operation that takes unit `()` as its input, we need to grab
                 // the QVoid.Instance value.
-                var bondLengths = H2BondLengths.Body.Invoke(QVoid.Instance);
+                var bondLengths = H2BondLengths.__Body__.Invoke(QVoid.Instance);
```

Comment:
It would be better to replace these with the sim.Run invocations - any reason why this should be invoked this way?

Reply:
I don't think `sim.Run` works here, but I updated the sample to use `op.Run(sim, ...)`.